### PR TITLE
15 minute path to nirvana

### PR DESCRIPTION
**Pull request by YOU54F**

### CODE REVIEW

**Comment by mefellows** [comment]
This dropped off my radar Saf - let's chat next week and see if we can get this to see the light of day!

I love the idea, I think we need to review the table of contents and flow through the docs though so it's not too overwhelming. Also I think we need to consider dropping the "nirvana" terminoogy.

**Comment by YOU54F** [comment]
Thanks buddy. I was chatting to Mike about this after his work on https://github.com/mikegeeves/pact-examples/tree/first

I'll copy-pasta the brain dump here and we can mull over it next week :)

If we have a way to show the same test across every language, we can just pull in the entire file by remote with a one liner.
We can think more about the user side presentation of telling them a story of how to tie it together.
There is always a good amount, if we aren’t thinking about re-use on the 5 minute getting started and the path to pact nirvana

- https://deploy-preview-189--docs-pact-io.netlify.app/15-minute-path-to-nirvana
- https://deploy-preview-189--docs-pact-io.netlify.app/pact_broker/samples_provider_verification_provider_changed#verifying-pacts-where-the-consumer-is-a-mobile-application

The latter shows multi code tabs but stored in the docs, which are hard to test.

We can use something like this to walk people though the repo for stand out steps

- https://techcommunity.microsoft.com/t5/educator-developer-blog/codetour-vscode-extension-allows-you-to-produce-interactive/ba-p/1274297

Includes markdown and runnable snippets

Allowing us to highlight specific areas of the code of interest maybe specific to a framework

And then we have killercodas, that if they run the same steps use in GitHub actions, they can be run with act, or locally on someone’s machine with code tour.

We can also reuse the single set of workshop docs, for the website and katacodas.

As shown with

- https://docs.pact.io/plugins/workshops/create-a-plugin/intro
- https://killercoda.com/pactflow/scenario/create-a-plugin

Basically don’t be blocked by an approach, what you think might work, might work. If it doesn’t we will find out over time. Having the same features tested cross language, even for the simplest case, provides the smallest regression suite we can, if that then gets triggered by a chance from the rust core, as a canary, we get getting safe reliable feedback.
If it’s useful we can get others into the fold. As we have tested code, we could build vscode snippets, so although we don’t give someone a full boilerplate code base they can run, they have something they can do into their code at their fingertips

**Comment by mefellows** [comment]
> Having the same features tested cross language, even for the simplest case, provides the smallest regression suite we can, if that then gets triggered by a chance from the rust core, as a canary, we get getting safe reliable feedback.

I think this is where the BDD suite might intersect, but I think this suggestion is the simplest and easiest possible first step.

I'm not sure I'm following how this guide might fit alongside the Nirvana guide and the 5 minute one? It feels like that might be a bit confusing/overwhelming?
